### PR TITLE
DOCS-6039 Embed Assignment - Getting Started w/ Session Replay

### PR DESCRIPTION
**Pull request by mariogerardi**
### Motivation
DOCS-6039

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.
- [ ] Check that `cache_enabled` is set to `true` in the `pull_config_preview.yaml` file

### CODE REVIEW

File: content/en/getting_started/session_replay/_index.md [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+
+Session Replay is available in the RUM Browser SDK. To start collecting data for Session Replay, set up Datadog RUM Browser Monitoring by creating a RUM application, generating a client token generation, and initializing the RUM Browser SDK. 
+
+For further instructions on collecting Session Replay data, follow the [RUM Getting Started guide][2] for your application. 
+
+For setup in mobile environments, see [Mobile Session Replay][3].
+
+## Finding particular Session Replays
+
+Once you're collecting Session Replay data, select **Session Replay available** in the [RUM Explorer][4] to see all sessions with a replay attached to it. From here, you can visualize the data as a **List**, **Timeseries**, or other format.
```
Comment (alai97) [inline content/en/getting_started/session_replay/_index.md:38]:
```suggestion
Once you're collecting Session Replay data, select **Session Replay available** in the [RUM Explorer][4] to see all sessions with a replay attached to it. You can visualize this data as a **List**, **Timeseries**, **Top List**, **Table**, **Distribution**, **Geomap**, **Funnel**, **Tree Map**, or **Pie Chart**. 
```

File: content/en/getting_started/session_replay/_index.md [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+
+For further instructions on collecting Session Replay data, follow the [RUM Getting Started guide][2] for your application. 
+
+For setup in mobile environments, see [Mobile Session Replay][3].
+
+## Finding particular Session Replays
+
+Once you're collecting Session Replay data, select **Session Replay available** in the [RUM Explorer][4] to see all sessions with a replay attached to it. From here, you can visualize the data as a **List**, **Timeseries**, or other format. 
+
+{{< img src="/getting_started/session_replay/session-replay-available.png" alt="Session Replay available button, as well as visualization options">}}
```
Comment (alai97) [inline content/en/getting_started/session_replay/_index.md:40]:
```suggestion
{{< img src="/getting_started/session_replay/session-replay-available.png" alt="Session Replay available button, as well as visualization options" style="width:100%" >}}
```

File: content/en/getting_started/session_replay/_index.md [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+
+### Facets
+
+You can filter Sessions using facets. Filtering by [facet][5] is helpful if you are searching for specific information, such as a particular user or device type. 
+
+### Views
+
+You are also able to filter by [view][6]. This can be helpful in the case that you know where the issue lies.
+
+{{< img src="/getting_started/session_replay/facets-views.png" alt="Filtering by facet and view">}}
```
Comment (alai97) [inline content/en/getting_started/session_replay/_index.md:50]: Looks like this image belongs in the ### Facets section.

File: content/en/getting_started/session_replay/_index.md [hunk 4]
```diff
@@ -0,0 +1,121 @@
...
+
+{{< img src="/getting_started/session_replay/session-replay-available.png" alt="Session Replay available button, as well as visualization options">}}
+
+### Facets
+
+You can filter Sessions using facets. Filtering by [facet][5] is helpful if you are searching for specific information, such as a particular user or device type. 
+
+### Views
+
+You are also able to filter by [view][6]. This can be helpful in the case that you know where the issue lies.
```
Comment (alai97) [inline content/en/getting_started/session_replay/_index.md:48]: I'd recommend adding an image that captures the Views dropdown menu or the My View button like: 
Screenshot 2023-09-12 at 8 34 19 AM

File: content/en/getting_started/session_replay/_index.md [hunk 5]
```diff
@@ -0,0 +1,121 @@
...
+
+{{< img src="/getting_started/session_replay/facets-views.png" alt="Filtering by facet and view">}}
+
+## User Journey
```
Comment (alai97) [inline content/en/getting_started/session_replay/_index.md:52]:
```suggestion
## Examine a user journey
```

File: content/en/getting_started/session_replay/_index.md (hunk 3 above)
Comment (alai97) [reply content/en/getting_started/session_replay/_index.md:50]:
```suggestion
{{< img src="/getting_started/session_replay/facets-views.png" alt="Filtering by facet and view" style="width:100%" >}}
```

File: content/en/getting_started/session_replay/_index.md [hunk 6]
```diff
@@ -0,0 +1,121 @@
...
+
+The **User Journey** functions as an event timeline, and is tracked on the right side of the page. Starting from the beginning of a replay, you can watch everything that a particular user did. You can navigate to any moment in the User Journey by selecting it.
```
Comment (alai97) [inline content/en/getting_started/session_replay/_index.md:54]:
```suggestion
The **User Journey** functions as an event timeline, and is tracked on the right side of the page. Starting from the beginning of a replay, you can watch everything that a particular user did. You can navigate to any moment in the user journey by clicking on an event.
```

File: content/en/getting_started/session_replay/_index.md [hunk 7]
```diff
@@ -0,0 +1,121 @@
...
+
+{{< img src="/getting_started/session_replay/user-journey.png" alt="Panel with User Journey">}}
+
+You are able to filter the User Journey by **View**, **Action**, **Error**, and **Frustration Signal** by selecting **Events**. 
+
+{{< img src="/getting_started/session_replay/views-actions-errors.png" alt="Panel that allows for filtering by view, action, error, and frustration signal">}}
```
Comment (alai97) [inline content/en/getting_started/session_replay/_index.md:60]:
```suggestion
{{< img src="/getting_started/session_replay/views-actions-errors.png" alt="Panel that allows for filtering by view, action, error, and frustration signal" style="width:100%" >}}
```

File: content/en/getting_started/session_replay/_index.md [hunk 8]
```diff
@@ -0,0 +1,121 @@
...
+
+You are also able to filter by [view][6]. This can be helpful in the case that you know where the issue lies.
+
+{{< img src="/getting_started/session_replay/facets-views.png" alt="Filtering by facet and view">}}
+
+## User Journey
+
+The **User Journey** functions as an event timeline, and is tracked on the right side of the page. Starting from the beginning of a replay, you can watch everything that a particular user did. You can navigate to any moment in the User Journey by selecting it.
+
+{{< img src="/getting_started/session_replay/user-journey.png" alt="Panel with User Journey">}}
```
Comment (alai97) [inline content/en/getting_started/session_replay/_index.md:56]:
```suggestion
{{< img src="/getting_started/session_replay/user-journey.png" alt="Panel with User Journey" style="width:100%" >}}
```

File: content/en/getting_started/session_replay/_index.md [hunk 9]
```diff
@@ -0,0 +1,121 @@
...
+
+While hovering over a particular time or view in the User Journey, selecting **Details** opens a side panel where you can examine specific Core Web Vitals of the page, as well as other pertinent information, without leaving the replay.
+
+{{< img src="/getting_started/session_replay/details-panel.png" alt="Panel with additional details">}}
+
+From this page, you can also expand the Performance Waterfall by selecting **Expand Waterfall** for more detailed information.
+
+{{< img src="/getting_started/session_replay/performance-waterfall.png" alt="Expanded performance waterfall">}}
+
+## Developer Tools
```
Comment (alai97) [inline content/en/getting_started/session_replay/_index.md:70]:
```suggestion
## Troubleshoot using Developer Tools
```

File: content/en/getting_started/session_replay/_index.md [hunk 10]
```diff
@@ -0,0 +1,121 @@
...
+
+Selecting **Dev Tools** enables you to view various aspects of your application's performance, console logs and errors associated with the view, and any application or user attributes regarding the replay.
```
Comment (alai97) [inline content/en/getting_started/session_replay/_index.md:72]:
```suggestion
Session Replay's [Browser Developer Tools][8] enable you to view various aspects of your application's performance, console logs, and errors associated with the view, and any application or user attributes regarding the replay. 
```

File: content/en/getting_started/session_replay/_index.md [hunk 11]
```diff
@@ -0,0 +1,121 @@
...
+
+{{< img src="/getting_started/session_replay/performance-waterfall.png" alt="Expanded performance waterfall">}}
+
+## Developer Tools
+
+Selecting **Dev Tools** enables you to view various aspects of your application's performance, console logs and errors associated with the view, and any application or user attributes regarding the replay. 
+
+{{< img src="/getting_started/session_replay/dev-tools.png" alt="Dev tools console">}}
+
+You can view the **Resources** and **Traces** associated with a view through using the tabs.
```
Comment (alai97) [inline content/en/getting_started/session_replay/_index.md:76]: I would separate these two sentences and have one screenshot per sentence. Also, I would add an anchor link to the APM integration section by using `[word](#apm-integration)`.

File: content/en/getting_started/session_replay/_index.md [hunk 12]
```diff
@@ -0,0 +1,121 @@
...
+
+From this page, you can also expand the Performance Waterfall by selecting **Expand Waterfall** for more detailed information.
+
+{{< img src="/getting_started/session_replay/performance-waterfall.png" alt="Expanded performance waterfall">}}
+
+## Developer Tools
+
+Selecting **Dev Tools** enables you to view various aspects of your application's performance, console logs and errors associated with the view, and any application or user attributes regarding the replay. 
+
+{{< img src="/getting_started/session_replay/dev-tools.png" alt="Dev tools console">}}
```
Comment (alai97) [inline content/en/getting_started/session_replay/_index.md:74]:
```suggestion
{{< img src="/getting_started/session_replay/dev-tools.png" alt="Dev tools console" style="width:100%" >}}
```

File: content/en/getting_started/session_replay/_index.md [hunk 13]
```diff
@@ -0,0 +1,121 @@
...
+
+You can view the **Resources** and **Traces** associated with a view through using the tabs.
+
+{{< img src="/getting_started/session_replay/resources.png" alt="Resources associated with the view">}}
```
Comment (alai97) [inline content/en/getting_started/session_replay/_index.md:78]:
```suggestion
{{< img src="/getting_started/session_replay/resources.png" alt="Resources associated with the view" style="width:100%" >}}
```

File: content/en/getting_started/session_replay/_index.md [hunk 14]
```diff
@@ -0,0 +1,121 @@
...
+
+{{< img src="/getting_started/session_replay/traces.png" alt="Traces associated with the view">}}
```
Comment (alai97) [inline content/en/getting_started/session_replay/_index.md:80]:
```suggestion
{{< img src="/getting_started/session_replay/traces.png" alt="Traces associated with the view" style="width:100%" >}}
```

File: content/en/getting_started/session_replay/_index.md [hunk 15]
```diff
@@ -0,0 +1,121 @@
...
+
+You can also view session metadata in the **Attributes** tab.
+
+{{< img src="/getting_started/session_replay/attributes.png" alt="Attributes tab of the Dev tools console">}}
```
Comment (alai97) [inline content/en/getting_started/session_replay/_index.md:84]:
```suggestion
{{< img src="/getting_started/session_replay/attributes.png" alt="Attributes tab of the Dev tools console" style="width:100%" >}}
```

File: content/en/getting_started/session_replay/_index.md [hunk 16]
```diff
@@ -0,0 +1,121 @@
...
+
+## Integrations
```
Comment (alai97) [inline content/en/getting_started/session_replay/_index.md:86]:
```suggestion
## Datadog integrations
```

File: content/en/getting_started/session_replay/_index.md [hunk 17]
```diff
@@ -0,0 +1,121 @@
...
+
+Session Replay is seamlessly integrated with your application's metrics, traces, and logs, giving you helpful context for debugging issues. Utilizing the APM and Error Tracking integrations alongside Session Replay enables you to investigate the root cause of user-facing issues, regardless of where they originate in your stack.
```
Comment (alai97) [inline content/en/getting_started/session_replay/_index.md:88]:
```suggestion
Session Replay integrates with your application's metrics, traces, and logs to give you helpful context for debugging issues. Utilizing the APM and Error Tracking integrations alongside Session Replay enables you to investigate the root cause of user-facing issues, regardless of where they originate in your stack.
```

File: content/en/getting_started/session_replay/_index.md [hunk 18]
```diff
@@ -0,0 +1,121 @@
...
+
+### APM integration
+
+Combining Session Replay with APM traces and metrics enables you to receive end-to-end visibility across frontend and backend issues, allowing you to see how code and infrastructure are impacting your user experience.
```
Comment (alai97) [inline content/en/getting_started/session_replay/_index.md:92]:
```suggestion
Combining Session Replay with [APM traces][9] enables you to receive end-to-end visibility across frontend and backend issues, and see how code and infrastructure are impacting your user experience.
```

File: content/en/getting_started/session_replay/_index.md [hunk 19]
```diff
@@ -0,0 +1,121 @@
...
+
+Within the Traces tab, select **View Trace in APM** to see more detailed information, including errors and logs associated with the trace. 
+
+{{< img src="/getting_started/session_replay/APM.png" alt="APM page with more detailed information">}}
```
Comment (alai97) [inline content/en/getting_started/session_replay/_index.md:96]:
```suggestion
{{< img src="/getting_started/session_replay/APM.png" alt="APM page with more detailed information" style="width:100%" >}}
```

File: content/en/getting_started/session_replay/_index.md [hunk 20]
```diff
@@ -0,0 +1,121 @@
...
+
+### Error Tracking integration
+
+The Error Tracking feature is helpful in debugging issues and getting to the root cause. You can receive alerts for an error, see the exact line of code that caused it, and pivot to view a user session that encountered the error all from within the Datadog App.
```
Comment (alai97) [inline content/en/getting_started/session_replay/_index.md:100]:
```suggestion
The [Error Tracking integration][10] is helpful in debugging issues and getting to the root cause. You can receive alerts for an error, see the exact line of code that caused it, and pivot to view a user session that encountered the error on the Datadog site.
```

File: content/en/getting_started/session_replay/_index.md [hunk 21]
```diff
@@ -0,0 +1,121 @@
...
+
+Within the Errors tab, selecting a specific error expands the view to display the time the error occurred, as well as the error message. Clicking **Issue in error tracking** opens a panel with more detail surrounding the error, as well as any attributes associated with the session.
```
Comment (alai97) [inline content/en/getting_started/session_replay/_index.md:102]:
```suggestion
In the **Errors** tab, selecting a specific error expands the view to display the time the error occurred, as well as the error message. Clicking **Issue in error tracking** opens a panel with more detail surrounding the error, as well as any attributes associated with the session.
```

File: content/en/getting_started/session_replay/_index.md [hunk 22]
```diff
@@ -0,0 +1,121 @@
...
+
+{{< img src="/getting_started/session_replay/error-tracking.png" alt="Error tracking panel">}}
+
+## What's next?
+
+You can share the replay with your team by selecting the **Share** dropdown at the top of the page. You can start the replay at a specific time, to direct your team's attention to a particular time and view of the Replay.
+
+{{< img src="/getting_started/session_replay/share.png" alt="Share your replay pop-up">}}
```
Comment (alai97) [inline content/en/getting_started/session_replay/_index.md:110]:
```suggestion
{{< img src="/getting_started/session_replay/share.png" alt="Share your replay pop-up" style="width:100%" >}}
```

File: content/en/getting_started/session_replay/_index.md [hunk 23]
```diff
@@ -0,0 +1,121 @@
...
+
+{{< img src="/getting_started/session_replay/APM.png" alt="APM page with more detailed information">}}
+
+### Error Tracking integration
+
+The Error Tracking feature is helpful in debugging issues and getting to the root cause. You can receive alerts for an error, see the exact line of code that caused it, and pivot to view a user session that encountered the error all from within the Datadog App.
+
+Within the Errors tab, selecting a specific error expands the view to display the time the error occurred, as well as the error message. Clicking **Issue in error tracking** opens a panel with more detail surrounding the error, as well as any attributes associated with the session.
+
+{{< img src="/getting_started/session_replay/error-tracking.png" alt="Error tracking panel">}}
```
Comment (alai97) [inline content/en/getting_started/session_replay/_index.md:104]:
```suggestion
{{< img src="/getting_started/session_replay/error-tracking.png" alt="Error tracking panel" style="width:100%" >}}
```

**Review by alai97 — CHANGES_REQUESTED** [review]
Some suggestions!

File: content/en/getting_started/session_replay/_index.md [hunk 24]
```diff
@@ -0,0 +1,121 @@
+---
+title: Getting Started with Session Replay
+kind: documentation
+further_reading:
+    - link: 'https://docs.datadoghq.com/real_user_monitoring/session_replay/'
+      tag: 'Documentation'
+      text: 'Session Replay'
+    - link: 'https://docs.datadoghq.com/real_user_monitoring/session_replay/mobile/'
+      tag: 'Documentation'
```
Comment (alai97) [inline content/en/getting_started/session_replay/_index.md:9]: For these docs links + the Troubleshooting link below, you can leave them as `/real_user_monitoring/session_replay/`, but for links outside of the documentation repo, they have to be absolute (full) URLs!

File: content/en/getting_started/session_replay/_index.md [hunk 25]
```diff
@@ -0,0 +1,121 @@
+---
+title: Getting Started with Session Replay
```
[the remark this answers is not in the document]
Comment (alai97) [reply content/en/getting_started/session_replay/_index.md:2]: One more entry in the partial list of the Getting Started page too: https://docs-staging.datadoghq.com/mario/sr-gs/getting_started/!

**Review by alai97 — APPROVED** [review]
Looks good to me pending my one follow up suggestion about updating the partial list!

File: content/en/getting_started/session_replay/_index.md [hunk 26]
```diff
@@ -0,0 +1,140 @@
...
+2. Generate a **Client Token**. 
+3. Initialize the RUM Browser SDK.
+
+Until Datadog starts receiving data, your application appears as `pending` on the **RUM Applications** page.
+
+For more detailed instructions on collecting Session Replay data, follow the [RUM Getting Started guide][2] for your application. For setup in mobile environments, see [Mobile Session Replay][3].
+
+## Find particular Session Replays
+
+Once you're collecting Session Replay data, navigate to the [RUM Explorer][4] and select **Session Replay available** to see all sessions with a replay attached to it. You can visualize this data as a **List**, **Timeseries**, **Top List**, **Table**, **Distribution**, **Geomap**, **Funnel**, **Tree Map**, or **Pie Chart**.
```
[the remark this answers is not in the document]
Comment (kayayarai) [reply content/en/getting_started/session_replay/_index.md:44]: In fact, we should be more opinionated in the GS. Create a scenario, like, "Suppose you've been told a customer with user ID `kelly.example@example.com` was experiencing problems" ... and tell them how to use filters/facets to find that session. "Or maybe you've created a Saved View (link to doc about saved views) that shows you all Sessions that encounter a certain error type. Open that Saved view"

File: content/en/getting_started/session_replay/_index.md [hunk 27]
```diff
@@ -0,0 +1,140 @@
...
+
+{{< img src="/getting_started/session_replay/session-replay-available.png" alt="Session Replay available button, as well as visualization options" style="width:100%" >}}
+
+You can filter Sessions using **facets**. Filtering by [facet][5] is helpful if you are searching for specific information, such as a particular user or device type. 
+
+{{< img src="/getting_started/session_replay/facets-views.png" alt="Filtering by facet" style="width:100%" >}}
+
+You are also able to filter by [view][6], as well as customize and pin views. This can be helpful in the case that you know where the issue lies.
+
+{{< img src="/getting_started/session_replay/pinned-views.png" alt="Views dropdown" style="width:100%" >}}
```
Comment (kayayarai) [inline content/en/getting_started/session_replay/_index.md:54]: If you be more scenario-driven as I have suggested above, you can skip this general stuff that is already covered in the RUM explorer docs, and just get to looking at Session Replays

File: content/en/getting_started/session_replay/_index.md [hunk 28]
```diff
@@ -0,0 +1,140 @@
...
+
+## Examine a user journey
+
+The **User Journey** functions as an event timeline, and is tracked on the right side of the page. Starting from the beginning of a replay, you can watch everything that a particular user did. You can navigate to any moment in the user journey by clicking on an event.
```
Comment (kayayarai) [inline content/en/getting_started/session_replay/_index.md:58]: you haven't mentioned here how to start or move around the SR playback, which you should. They can either click points in the user journey list, or they can click the play button on the SR "video"

Comment (kayayarai) [inline content/en/getting_started/session_replay/_index.md:58]:
```suggestion
The **User Journey** is an event timeline on the right side of the page. The session replay looks like a video on the left, with standard video navigation tools. Start the replay from the beginning by clicking play, and watch everything that a particular user did. You can navigate to any moment in the user journey by clicking on an event in the list.
```

File: content/en/getting_started/session_replay/_index.md [hunk 29]
```diff
@@ -0,0 +1,140 @@
...
+
+{{< img src="/getting_started/session_replay/user-journey.png" alt="Panel with User Journey" style="width:100%" >}}
+
+You are able to filter the User Journey by **View**, **Action**, **Error**, and **Frustration Signal** by selecting **Events**.
```
Comment (kayayarai) [inline content/en/getting_started/session_replay/_index.md:62]:
```suggestion
Filter the user journey list by selecting or clearing types of events: 
```

File: content/en/getting_started/session_replay/_index.md [hunk 30]
```diff
@@ -0,0 +1,140 @@
...
+
+The **User Journey** functions as an event timeline, and is tracked on the right side of the page. Starting from the beginning of a replay, you can watch everything that a particular user did. You can navigate to any moment in the user journey by clicking on an event.
+
+{{< img src="/getting_started/session_replay/user-journey.png" alt="Panel with User Journey" style="width:100%" >}}
+
+You are able to filter the User Journey by **View**, **Action**, **Error**, and **Frustration Signal** by selecting **Events**. 
+
+{{< img src="/getting_started/session_replay/views-actions-errors.png" alt="Panel that allows for filtering by view, action, error, and frustration signal" style="width:100%" >}}
+
+While hovering over a particular time or view in the user journey, selecting **Details** opens a side panel where you can examine specific Core Web Vitals of the page, as well as other pertinent information, without leaving the replay.
```
Comment (kayayarai) [inline content/en/getting_started/session_replay/_index.md:66]:
```suggestion
While hovering over a particular time or view in the user journey, select **Details** to examine Core Web Vitals and other pertinent information, without leaving the replay.
```

File: content/en/getting_started/session_replay/_index.md [hunk 31]
```diff
@@ -0,0 +1,140 @@
...
+
+{{< img src="/getting_started/session_replay/details-panel.png" alt="Panel with additional details">}}
+
+From this page, you can also expand the Performance Waterfall by selecting **Expand Waterfall** for more detailed information.
```
Comment (kayayarai) [inline content/en/getting_started/session_replay/_index.md:70]:
```suggestion
From the Details page, you can expand the waterfall view for more detailed information.
```

File: content/en/getting_started/session_replay/_index.md [hunk 32]
```diff
@@ -0,0 +1,140 @@
...
+
+{{< img src="/getting_started/session_replay/performance-waterfall.png" alt="Expanded performance waterfall">}}
+
+## Troubleshoot using Developer Tools
+
+Session Replay's [Browser Developer Tools][8] enable you to view various aspects of your application's performance, console logs and errors associated with the view, and any application or user attributes regarding the replay.
```
Comment (kayayarai) [inline content/en/getting_started/session_replay/_index.md:76]:
```suggestion
Open Session Replay's [browser developer tools][8] to explore your application's performance, console logs, errors, and application or user attributes associated with the current replay. 
```

File: content/en/getting_started/session_replay/_index.md [hunk 33]
```diff
@@ -0,0 +1,140 @@
...
+
+Session Replay's [Browser Developer Tools][8] enable you to view various aspects of your application's performance, console logs and errors associated with the view, and any application or user attributes regarding the replay. 
+
+{{< img src="/getting_started/session_replay/dev-tools.png" alt="Dev tools console" style="width:100%" >}}
+
+You can view any **Resources** associated with a view by selecting the Resources tab.
+
+{{< img src="/getting_started/session_replay/resources.png" alt="Resources associated with the view" style="width:100%" >}}
+
+**Traces** associated with a view can be found in the [Traces tab](#tracing-with-the-apm-integration).
```
Comment (kayayarai) [inline content/en/getting_started/session_replay/_index.md:84]: These are very close to "click print to print" type documentation 😂 . WHY would someone look at resources? what kind of info will they find there to help them troubleshoot? What should they look for? Ditto with Traces

File: content/en/getting_started/session_replay/_index.md [hunk 34]
```diff
@@ -0,0 +1,140 @@
...
+
+{{< img src="/getting_started/session_replay/traces.png" alt="Traces associated with the view" style="width:100%" >}}
+
+Session metadata is populated in the **Attributes** tab.
```
Comment (kayayarai) [inline content/en/getting_started/session_replay/_index.md:88]: Provide an example of what might be a useful piece of info on the Attributes tab.

File: content/en/getting_started/session_replay/_index.md [hunk 35]
```diff
@@ -0,0 +1,140 @@
...
+
+Session Replay integrates with your application's metrics, traces, and logs to give you helpful context for debugging issues. Utilizing the APM and Error Tracking integrations alongside Session Replay enables you to investigate the root cause of user-facing issues, regardless of where they originate in your stack.
+
+### Tracing with the APM integration
+
+Combining Session Replay with [APM traces][9] enables you to receive end-to-end visibility across frontend and backend issues, and see how code and infrastructure are impacting your user experience.
+
+Within the Traces tab, select **View Trace in APM** to see more detailed information, including errors and logs associated with the trace. 
+
+{{< img src="/getting_started/session_replay/APM.png" alt="APM page with more detailed information" style="width:100%" >}}
```
Comment (kayayarai) [inline content/en/getting_started/session_replay/_index.md:102]: I think if you provide a little more specifics of what a person might find in a trace, as I commented on in lines 80-84, you can get rid of this section entirely.

File: content/en/getting_started/session_replay/_index.md [hunk 36]
```diff
@@ -0,0 +1,140 @@
...
+
+### Investigating errors with the Error Tracking integration
+
+The [Error Tracking integration][10] is helpful in debugging issues and getting to the root cause. You can receive alerts for an error, see the exact line of code that caused it, and pivot to view a user session that encountered the error on the Datadog site.
+
+In the **Errors** tab, selecting a specific error expands the view to display the time the error occurred, as well as the error message. Clicking **Issue in error tracking** opens a panel with more detail surrounding the error, as well as any attributes associated with the session.
```
Comment (kayayarai) [inline content/en/getting_started/session_replay/_index.md:108]:
```suggestion
In the **Errors** tab, select an error to see the time the error occurred and the message. Click **Issue in error tracking** to see more detail and attributes associated with the session.
```

File: content/en/getting_started/session_replay/_index.md [hunk 37]
```diff
@@ -0,0 +1,140 @@
...
+
+{{< img src="/getting_started/session_replay/error-tracking.png" alt="Error tracking panel" style="width:100%" >}}
+
+## What's next?
+
+### Create Browser Tests from Session Replays
```
Comment (kayayarai) [inline content/en/getting_started/session_replay/_index.md:114]:
```suggestion
### Create Synthetic browser tests from Session Replays
```
The Synthetic part is important -- Synthetic tests are automated tests that Datadog simulates on real running code.

File: content/en/getting_started/session_replay/_index.md [hunk 38]
```diff
@@ -0,0 +1,140 @@
...
+
+You can [create a browser test][11] from the exact sequence of steps your users went through in a Session Replay. Navigate to the [RUM Explorer][4] and select a session with an available Session Replay that you want to create a browser test from. Click **Generate Synthetic Browser Test** above the event timeline.
```
Comment (kayayarai) [inline content/en/getting_started/session_replay/_index.md:116]:
```suggestion
You can [create a synthetic browser test][11] from the exact sequence of steps your users went through in a Session Replay. Datadog runs synthetic tests on an automated schedule you define, to simulate user behavior, reporting failing tests to you, without your users having to encounter the problem again.

To capture a session replay in a synthetic browser test, click **Generate Synthetic Browser Test** above the event timeline.
```

Comment (kayayarai) [reply content/en/getting_started/session_replay/_index.md:116]: ... and maybe tell them to go finish configuring the synth test in the Synthetic Test product (and link to those docs).

**Review by kayayarai — CHANGES_REQUESTED** [review]
A bunch of comments! Let me know if any of them don't make sense or if you're not sure what to do with them.

File: content/en/getting_started/session_replay/_index.md (hunk 34 above)
Comment (mariogerardi) [reply content/en/getting_started/session_replay/_index.md:90]: I just got rid of this. The only thing that is found under the Attributes tab is session/user metadata, and it felt self-explanatory.

File: content/en/getting_started/session_replay/_index.md [hunk 39]
```diff
@@ -0,0 +1,139 @@
...
+
+Once you're collecting Session Replay data, navigate to the [RUM Explorer][4] and select **Session Replay available** to see all sessions with a replay attached to it. You can visualize this data as a **List**, **Timeseries**, or other format.
+
+{{< img src="/getting_started/session_replay/session-replay-available.png" alt="Session Replay available button, as well as visualization options" style="width:100%" >}}
+
+Suppose you've been told that a customer experienced issues with your application on a mobile device. You can filter Sessions using **facets**. In this case, filtering by [facet][5] is helpful in searching for specific information, such as a particular user or device type.
+
+{{< img src="/getting_started/session_replay/facets-views.png" alt="Filtering by facet" style="width:100%" >}}
+
+Maybe you've created a [Saved View][6] that shows you all Sessions that contain the specific error that your customer encountered. This can be helpful in the case that you know where the issue lies.
```
Comment (kayayarai) [inline content/en/getting_started/session_replay/_index.md:54]:
```suggestion
Maybe you've created a [Saved View][6] that shows you all sessions that contain a specific error that your application generates. Bringing up this view is helpful when that you know where the issue lies, and want to see session replays of people encountering it.
```

File: content/en/getting_started/session_replay/_index.md [hunk 40]
```diff
@@ -0,0 +1,139 @@
...
+
+{{< img src="/getting_started/session_replay/pinned-views.png" alt="Views dropdown" style="width:100%" >}}
+
+## Examine a user journey
+
+The **User Journey** is an event timeline on the right side of the page. The session replay looks like a video on the left, with standard video navigation tools. Start the replay from the beginning by clicking play, and watch everything that a particular user did. You can navigate to any moment in the user journey by clicking on an event in the list. You can also track all the actions and errors occurring for each view by clicking **Session Breakdown**.
```
Comment (kayayarai) [inline content/en/getting_started/session_replay/_index.md:60]:
```suggestion
The session replay looks like a video on the left, with standard video navigation tools. Start the replay from the beginning by clicking play, and watch everything that a particular user did. 

The **User Journey** is an event timeline on the right side of the page. You can navigate to any moment in the user journey by clicking on an event in the list. You can also track all the actions and errors occurring for each view by clicking **Session Breakdown**.
```

File: content/en/getting_started/session_replay/_index.md [hunk 41]
```diff
@@ -0,0 +1,139 @@
...
+
+## Troubleshoot using Developer Tools
+
+Open Session Replay's [browser developer tools][8] to explore your application's performance, console logs, errors, and application or user attributes associated with the current replay. 
+
+{{< img src="/getting_started/session_replay/dev-tools.png" alt="Dev tools console" style="width:100%" >}}
+
+## Pivot to correlated data
+
+Session Replay integrates with your application's metrics, traces, and logs to give you helpful context for debugging issues. Utilizing the APM and Error Tracking integrations alongside Session Replay enables you to investigate the root cause of user-facing issues, regardless of where they originate in your stack.
```
Comment (kayayarai) [inline content/en/getting_started/session_replay/_index.md:87]:
```suggestion
Session Replay integrates with your application's metrics, traces, and logs to give you helpful context for debugging issues. Using APM and Error Tracking alongside Session Replay enables you to investigate the root cause of user-facing issues, regardless of where they originate in your stack.
```

File: content/en/getting_started/session_replay/_index.md [hunk 42]
```diff
@@ -0,0 +1,139 @@
...
+
+### Tracing with the APM integration
```
Comment (kayayarai) [inline content/en/getting_started/session_replay/_index.md:89]:
```suggestion
### Investigating request performance with APM traces
```

File: content/en/getting_started/session_replay/_index.md [hunk 43]
```diff
@@ -0,0 +1,139 @@
...
+
+Combining Session Replay with [APM traces][9] enables you to receive end-to-end visibility across frontend and backend issues, and see how code and infrastructure are impacting your user experience. Having full-stack traces can be helpful if you're unsure whether an error is occurring on the frontend or backend of your application.
```
Comment (kayayarai) [inline content/en/getting_started/session_replay/_index.md:91]:
```suggestion
The [APM traces][9] associated with a session replay give you end-to-end visibility across frontend and backend issues, and see how code and infrastructure are impacting your user experience. Having full-stack traces can be helpful if you're unsure whether an error is occurring on the frontend or backend of your application. 
```

File: content/en/getting_started/session_replay/_index.md [hunk 44]
```diff
@@ -0,0 +1,139 @@
...
+
+Select a replay with traces to view the browser request, as well as all the backend dependencies and services called upon to fulfill the request in a specific page.
+
+{{< img src="/getting_started/session_replay/traces-view.png" alt="Traces panel" style="width:100%" >}}
+
+Select **View Trace in APM** to see more detailed information, including errors and logs associated with the trace. 
+
+{{< img src="/getting_started/session_replay/APM.png" alt="APM page with more detailed information" style="width:100%" >}}
+
+### Investigating errors with the Error Tracking integration
```
Comment (kayayarai) [inline content/en/getting_started/session_replay/_index.md:101]:
```suggestion
### Investigating errors with Error Tracking
```

File: content/en/getting_started/session_replay/_index.md [hunk 45]
```diff
@@ -0,0 +1,139 @@
...
+
+The [Error Tracking integration][10] is helpful in debugging issues and getting to the root cause. You can receive alerts for an error, see the exact line of code that caused it, and pivot to view a user session that encountered the error on the Datadog site.
```
Comment (kayayarai) [inline content/en/getting_started/session_replay/_index.md:103]:
```suggestion
[Error Tracking][10] is helpful in debugging issues and getting to the root cause. You can receive alerts for an error occurring, see the exact line of code that caused it, and pivot to view a user session that encountered the error.
```

File: content/en/getting_started/session_replay/_index.md [hunk 46]
```diff
@@ -0,0 +1,139 @@
...
+## What's next?
+
+### Create Synthetic browser tests from Session Replays
+
+You can [create a synthetic browser test][11] from the exact sequence of steps your users went through in a Session Replay. Datadog runs synthetic tests on an automated schedule you define, to simulate user behavior, reporting failing tests to you without your users having to encounter the problem again.
+
+To capture a session replay in a synthetic browser test, click **Generate Synthetic Browser Test** above the event timeline.
+
+{{< img src="/getting_started/session_replay/browser-test.png" alt="Browser test creation popup window" style="width:100%" >}}
+
```
Comment (kayayarai) [inline content/en/getting_started/session_replay/_index.md:118]:
```suggestion

For more information about managing, running, and interpreting test results, read [Synthetic Browser Tests][12].

```

File: content/en/getting_started/session_replay/_index.md [hunk 47]
```diff
@@ -0,0 +1,139 @@
...
+[2]: /real_user_monitoring/#get-started
+[3]: /real_user_monitoring/session_replay/mobile/
+[4]: https://app.datadoghq.com/rum/sessions
+[5]: /real_user_monitoring/explorer/search/#facets
+[6]: /real_user_monitoring/explorer/saved_views/
+[7]: /real_user_monitoring/browser/#setup
+[8]: /real_user_monitoring/session_replay/developer_tools
+[9]: /real_user_monitoring/connect_rum_and_traces
+[10]: /real_user_monitoring/error_tracking/
+[11]: /synthetics/guide/rum-to-synthetics/
```
Comment (kayayarai) [inline content/en/getting_started/session_replay/_index.md:139]:
```suggestion
[11]: /synthetics/guide/rum-to-synthetics/
[12]: /synthetics/browser_tests/
```

**Review by kayayarai — CHANGES_REQUESTED** [review]

**Review by kayayarai — APPROVED** [review]

**Pull request merged**